consensus/src/core.rs

```diff
@@ -134,16 +135,21 @@ impl<Mempool: 'static + NodeMempool> Core<Mempool> {
     }
 
     async fn make_vote(&mut self, block: &Block) -> Option<Vote> {
+        // Update the preferred round.
+        self.update_preferred_round(block.qc.round);
```

Comment:
Preferred round is not really necessary in a 2-chain, we just have a locked round (high-qc) and a committed round